runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -8443,9 +8441,9 @@ makeMemSetGraph(TR::Compilation *c, int32_t ctrl)
    return tgt;
    }
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```